buildlib/jucx/jucx-build.yml

```diff
@@ -82,12 +85,20 @@ jobs:
           cp $(publicKey.secureFilePath)  ${{ parameters.gpg_dir }}/pubring.gpg
           cp $(privateKey.secureFilePath) ${{ parameters.gpg_dir }}/secring.gpg
           export GNUPGHOME=${{ parameters.gpg_dir }}
+
+          # GPG agent config
```

Comment:
Is this a reference config? Where is it from?

Reply:
Based on experimenting with various solutions recommended in online forums.